pkgs/desktops/gnome/extensions/update-extensions.py

```diff
@@ -48,28 +47,39 @@ def fetch_extension_data(uuid: str, version: str) -> Tuple[str, str]:
     uuid = uuid.replace("@", "")
     url: str = f"https://extensions.gnome.org/extension-data/{uuid}.v{version}.shell-extension.zip"
 
-    # Yes, we download that file three times:
+    # TODO remove when Vitals@CoreCoding.com version != 53, this extension has a missing manifest.json
+    if url == 'https://extensions.gnome.org/extension-data/VitalsCoreCoding.com.v53.shell-extension.zip':
+        url = 'https://extensions.gnome.org/extension-data/VitalsCoreCoding.com.v53.shell-extension_v1BI2FB.zip'
 
-    # The first time is for the maintainter, so they may have a personal backup to fix potential issues
+    # Yes, we download that file two times:
+
+    # The first time is for the maintainer, so they may have a personal backup to fix potential issues
```

Comment:
That part is commented out anyways for a while now, since the script is running rather stably. So maybe just remove the entire section around that comment?

Reply:
I agree it's not necessary. Also if you want to debug you can just get the dir out of the store.